packages/wrangler/e2e/versions.test.ts

```diff
@@ -76,15 +76,15 @@ describe("versions deploy", { timeout: TIMEOUT }, () => {
 			"Version ID:  00000000-0000-0000-0000-000000000000
 			Created:     TIMESTAMP
 			Author:      person@example.com
-			Source:      Unknown (version_upload)
-			Tag:         e2e-upload
-			Message:     Upload via e2e test
+			Source:      Upload
+			Tag:         -
+			Message:     -
 			Version ID:  00000000-0000-0000-0000-000000000000
 			Created:     TIMESTAMP
 			Author:      person@example.com
-			Source:      Upload
-			Tag:         -
-			Message:     -"
+			Source:      Unknown (version_upload)
+			Tag:         e2e-upload
+			Message:     Upload via e2e test"
```

Comment:
Looks like this order has been reversed. Was that intentional?

Reply:
Yes, version & deployment list are reversed - newest last (at the bottom)

We looked at the usage and it's very low so we're taking advantage of that + beta to make this change. This will make it much more inline with what users expect and easier to see the latest in terminals.